substratest/assets.py

```diff
@@ -1,9 +1,11 @@
 import abc
 import enum
-import dataclasses
 import re
 import time
 import typing
+import pydantic
+
+from inspect import isclass
 
 from . import errors, cfg
```

Comment:
The order is still incorrect:
- second group should include only pydantic
- third group should include only errors / cfg
- first group should include all the other libs (standard)

Reply:
Thanks! I just modified this 🙂